### ISSUE TITLE
Images not centered in some cases

### ISSUE THREAD

**Issue by amo13** [TryGhost/London#10]
Maybe I did something wrong during migration from one server to another, but images on my instance with this theme show uncentered in posts. See adaptief.de
When I activate the casper theme, none of the user uploaded images show up at all, only the unsplash ones - don't know if it could be related?

**Comment by rishabhgrg** [comment]
👋  Hey @amo13, this seems to more of an issue of how images were migrated than of theme, as normally images are correctly positioned with respect to theme. Please let me know if you have any usecase with which the issue can be reproduced and I can take a look at it again, closing this in the meanwhile.

**Comment by amo13** [comment]
I just updated ghost to the last version and the issue persists. If I change themes, I can see that the post image (the main image is the only one concerned) is only left-aligned in this theme. It is correctly centered with the casper theme (and with attila, editorial and massively). I zoomed out on the site and I took two screenshots for comparison. First one is london theme, second is casper:

![Screenshot from 2019-07-25 11-47-01](https://user-images.githubusercontent.com/6617661/61865942-8473eb80-aed4-11e9-981e-036935b04f57.png)

![Screenshot from 2019-07-25 11-58-46](https://user-images.githubusercontent.com/6617661/61865912-702fee80-aed4-11e9-8f39-4e9f11491f53.png)

**Comment by amo13** [comment]
Created a pull request for a fix.

**Comment by rishabhgrg** [comment]
Hey @amo13, thanks for the PR. I had another look at the issue and [PR](https://github.com/TryGhost/London/pull/20),  and I can understand why the feature images were not centered when you raised original issue. We had added a fix for centering images properly on larger viewports [here](https://github.com/TryGhost/London/commit/1055db6f4ce891a5a91cd39ef3cbb79f1653d613) a little later after your issue was raised which fixes this particular use-case, which should also work in properly centering the image. Looking at adaptief.de, it seems to be still using older css files from before this change, and updating theme to latest should also work. Closing the PR in favour of our existing code, thanks for the effort 😄
